{"name": "risk user -- @1.4 cases", "status": "passed", "steps": [{"name": "Given risk makes get request: /reconciliation/all_users_data/xlsx", "status": "passed", "start": 1610713914839, "stop": 1610713915403}, {"name": "Then check response of request: You do not have permission to perform this action. is False", "status": "passed", "start": 1610713915403, "stop": 1610713915562}], "parameters": [{"name": "url", "value": "/reconciliation/all_users_data/xlsx"}, {"name": "method", "value": "get"}, {"name": "bool", "value": "False"}], "start": 1610713914834, "stop": 1610713915562, "uuid": "2d2936f0-78b2-4903-a1bc-9944e28e55ab", "historyId": "ad308c72d421c1e6a04fe8711c9cc284", "fullName": "features/III-reconciliation_access.feature:risk user", "labels": [{"name": "severity", "value": "normal"}, {"name": "feature", "value": "reconciliation check access"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython3"}]}